src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -138,6 +138,9 @@ export const selectTotalProtocolFeeByAsset: Selector<
   quote ? getTotalProtocolFeeByAsset(quote) : undefined,
 )
 
+export const selectIsActiveQuoteMultiHop: Selector<ReduxState, boolean | undefined> =
+  createSelector(selectActiveQuote, quote => (quote ? quote?.steps.length > 1 : undefined))
```

Comment:
Do we want to consistently return a boolean here or is undefined intended to signify intent of the quote being possibly nullish?

Reply:
undefined means there is no active quote, meaning `isActiveQuoteMultiHop` is not defined 👍 useful for handling indeterminate state where we dont want app to behave as though there is 1 or 2 hops, but instead there is no quote at all